.github/file-paths.yaml

```diff
@@ -42,6 +42,13 @@ frontend_all: &frontend_all
   - *frontend_sources
   - *frontend_specs
 
+backend_ci: &backend_ci
+  - ".github/actions/prepare-frontend/**"
+  - ".github/actions/prepare-backend/**"
+  - ".github/actions/test-driver/**"
+  - ".github/workflows/backend.yml"
+  - ".github/workflows/drivers.yml"
+
```

Comment:
In theory, this can be broken down even further into `backend_ci` and `drivers_ci`. If someone wants to follow up with that change, please be my guest.